code/modules/paperwork/photocopier.dm

```diff
@@ -298,16 +304,19 @@
 			W = copy(W)
 		else if(istype(W, /obj/item/photo))
 			W = photocopy(W)
-		W.forceMove(p)
-		p.amount++
-	p.amount--
-	p.forceMove(get_turf(src))
-	p.update_icon()
-	p.icon_state = "paper_words"
-	p.name = bundle.name
-	p.pixel_y = rand(-8, 8)
-	p.pixel_x = rand(-9, 9)
-	return p
+		W.forceMove(P)
+		P.amount++
+	if(!P.amount)
+		qdel(P)
+		return null
+	P.amount--
+	P.forceMove(get_turf(src))
+	P.update_icon()
+	P.icon_state = "paper_words"
+	P.name = bundle.name
+	P.pixel_y = rand(-8, 8)
+	P.pixel_x = rand(-9, 9)
```

Comment:
Out of interest, why is it `9` here and `8` directly above?

Reply:
Maybe paper is longer than it is wider?

Reply:
You'll have to ask the guy before me